pandas/tests/io/test_excel.py

```diff
@@ -49,6 +49,21 @@ def ignore_xlrd_time_clock_warning():
         yield
 
 
+@contextlib.contextmanager
```

Comment:
Can you make this a fixture instead?

Reply:
I followed the example of `ignore_xlrd_time_clock_warning`, shall I turn that into a fixture as well? Also, `ignore_xlrd_time_clock_warning` and `ignore_openpyxl_unknown_extension_warning` are always used together, shall I combine them?

Reply:
I'd leave the xlrd one alone; that would make a nice follow up to consolidate